project-words.txt

```diff
@@ -23,6 +23,7 @@ autogenerating
 autohide
 Autolinks
 Bartosz
+bedly
```

Comment:
I'd prefer if your test labels did not introduce unknown project words

```ts
  it('reject bedly defined tags object options', () => {
```

What does bedly mean? Why did you choose this value?